rust/main/submitter/src/chain_tx_adapter/transaction.rs

```diff
@@ -0,0 +1,33 @@
+// TODO: re-enable clippy warnings
+#![allow(dead_code)]
+
+use hyperlane_core::H256;
+use uuid::Uuid;
+
+type PayloadId = Uuid;
+type SignerAddress = H256;
```

Comment:
I wonder if we can share this aliases with `payload_dispatcher` as well. `PayloadId` alias will be useful there.

And I wonder if we need a similar alias for identifier of a transaction?

Reply:
hm, sure I'll rename both `id` fields and export